CHANGELOG.md

```diff
@@ -2,9 +2,11 @@
 
 ## VNext
 
-## Version 2.20.0-beta1
+## Version 2.20
 - [Allow Control of Documents sampling for QuickPulse telemetry](https://github.com/microsoft/ApplicationInsights-dotnet/pull/2425)
 
+## Version 2.20.0-beta1
+- [Allow Control of Documents sampling for QuickPulse telemetry](https://github.com/microsoft/ApplicationInsights-dotnet/pull/2425)
```

Comment:
This section is repeated twice in GA and beta. Is this intentional?

Reply:
yes it was.  :)
normally i don't do this, but we're doing this release for our partner team and it's literally the only change that made it this milestone so i did it on a whim this time.